public/404.html

```diff
@@ -1,67 +1,213 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>The page you were looking for doesn't exist (404)</title>
+  <title>Page Not Found | Classroom</title>
   <meta name="viewport" content="width=device-width,initial-scale=1">
+  <link href='//fonts.googleapis.com/css?family=Open+Sans:300italic,400italic,600italic,700italic,800italic,400,300,600,700,800' rel='stylesheet' type='text/css'>
```

Comment:
`stylesheet_link_tag` may allow you to load app styles here. If not, we might just have to hardcode.